apps/acceptance/clock/regions/stopwatch.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+var PageRegion = require(
+    '../../../../shared/test/integration/helpers/pageRegion');
+
+function Stopwatch(client) {
+    PageRegion.call(this, client,
+        Stopwatch.prototype.selectors._stopwatchViewLocator);
+}
+
+Stopwatch.prototype = Object.create(PageRegion.prototype);
+Stopwatch.prototype.constructor =  Stopwatch;
+
+Stopwatch.prototype.selectors = {
+    _stopwatchViewLocator :
+    {'by':'id', 'locator': 'stopwatch-panel' },
+    _stopwatchTimeLocator : {'by': 'css selector',
+        'locator': '.stopwatch-time'},
+    _stopwatchStartLocator: {'by': 'css selector',
+        'locator': '#stopwatch-controls .stopwatch-start'},
+    _stopwatchPauseLocator: {'by': 'css selector',
+        'locator': '#stopwatch-controls .stopwatch-pause'},
+    _stopwatchResetLocator: {'by': 'css selector',
+        'locator': '#stopwatch-controls .stopwatch-reset'},
+    _stopwatchResumeLocator: {'by': 'css selector',
+        'locator': '#stopwatch-controls .stopwatch-resume'},
+    _stopwatchLapLocator: {'by': 'css selector',
+        'locator': '#stopwatch-controls .stopwatch-lap'}
+};
+
+Stopwatch.prototype.getCurrentTime = function() {
+    var currentTime = this.findElement(
+        Stopwatch.prototype.selectors._stopwatchTimeLocator);
+    return currentTime.text();
+};
+
+Stopwatch.prototype.tapStart = function() {
+    this.tap(Stopwatch.prototype.selectors._stopwatchStartLocator);
+};
+
+Stopwatch.prototype.tapPause = function() {
+    this.tap(Stopwatch.prototype.selectors._stopwatchPauseLocator);
+};
+
+Stopwatch.prototype.tapReset = function() {
+    this.tap(Stopwatch.prototype.selectors._stopwatchResetLocator);
+};
+
+Stopwatch.prototype.tapResume = function() {
+    this.tap(Stopwatch.prototype.selectors._stopwatchResumeLocator);
+};
+
+Stopwatch.prototype.tapLap = function() {
```

Comment:
I added this because the next test case will need it. Especially if we are going to be porting dogfood tests. Do you want me to remove it?